### PR TITLE
Added preprocess.Treadmill

### PR DESCRIPTION
**Pull request by jakereimer**
Load treadmill velocity and timestamps from behavior file (issue #95)

### CODE REVIEW

File: matlab/schemas/+preprocess/Treadmill.m [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            end
+            ballT = pipetools.ts2sec(dat.ball(:,2), packetLen);
+            tuple.treadmill_time = ballT;
+            
+            tuple.treadmill_raw = unwrap(dat.ball(:,1),2^31);
+            
+            bins100ms = ballT(1):.1:ballT(end);
+            distPerBin = [0 diff(interp1(ballT,tuple.treadmill_raw-tuple.treadmill_raw(1),bins100ms))]; % integrated distance per 100ms bin
+            
+            tuple.treadmill_cmsec_factor = .09; %(71.8168 cm circumference of 9inch wheel generates 8000 encoder counts = ~.009cm per count * 10 100ms bins per second)
```
Comment (fabiansinz) [inline matlab/schemas/+preprocess/Treadmill.m]: Can that differ between setups? If yes, what about putting that info in a Lookup table?

Comment (jakereimer) [inline matlab/schemas/+preprocess/Treadmill.m]: Good idea. Added lookup table experiment.TreadmillSpecs

Comment (fabiansinz) [inline matlab/schemas/+preprocess/Treadmill.m]: Do you plan to use the values in there to compute `tuple.treadmill_cmsec_factor`?

Comment (jakereimer) [inline matlab/schemas/+preprocess/Treadmill.m]: oops, forgot to include the updated `preprocess.Treadmill` in commit.

**Pull request merged**